spaceR/SpacerTests/SpaceRTests.swift

```diff
@@ -0,0 +1,39 @@
+////
+////  SpaceRTests.swift
+////  spaceR
+////
+////  Created by Kieran Woodrow on 2022/02/25.
+////
+//
+//import Foundation
+//
+//
+//import XCTest
+//@testable import spaceR
+//
+//class SpaceRTests: XCTestCase {
```

Comment:
So you can leave these classes uncommented, we don't really like committing commented code.

Reply:
I commented it out because i can't figure out why it wont find XCTest module.. Been looking online and it says to make sure @testable import "modulename" is the right modulename and it is.. So just need to figure out why it's not picking it up

Reply:
Fixed this issue 🤙🏼